lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -195,22 +199,28 @@ def _check_summing_mismatch(track, sampling_width):
             else:
                 return RuntimeWarning(wrong_kymo_warning)
     except IndexError:
-        return RuntimeWarning(wrong_kymo_warning)
+        raise ValueError(wrong_kymo_warning)
 
 
-def import_kymotrackgroup_from_csv(filename, kymo, channel, delimiter=";"):
-    """Import a KymoTrackGroup from a csv file.
+def load_tracks(filename, kymo, channel, delimiter=";"):
+    """Loads a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup` from a csv file.
 
     The file format contains a series of columns as follows:
     track index, time (pixels), coordinate (pixels), time (optional), coordinate (optional),
     sampled_counts (optional), minimum length
 
+    .. note::
+
+        If you cropped the kymograph prior to tracking, you will have to make sure that you crop
+        the kymograph you supply to this function in the same way.
```

Comment:
It is indeed. The list is not exhaustive. I will phrase it as processing and make clear that the list may not be exhaustive (because we may forget to update the docstring when additional processing is added).